code/controllers/subsystem/job.dm

```diff
@@ -64,6 +64,12 @@ SUBSYSTEM_DEF(job)
 	/// Dictionary that maps job priorities to low/medium/high. Keys have to be number-strings as assoc lists cannot be indexed by integers. Set in setup_job_lists.
 	var/list/job_priorities_to_strings
 
+	/// List of job config datum singletons.
+	var/static/list/job_config_datum_singletons = list()
+
+	/// List of the types of things that we can configure on a job (practically, the key for the singletons).
+	var/static/list/job_config_datum_configurables = list()
```

Comment:
as said before, these should be one rather then two
also why are they static? to persist through subsystem rebuild?

Reply:
yeah this is mirroring what SSvote does with its vote singletons since we don't need to waste time newing the datums again

Reply:
why not just copy them over on subsystem restore?

Reply:
i looked at SSvote again and noticed that it didn't actually use static lists and that i was hallucinating that so i'll just make 'em normal